tests/BrowscapTest/Data/DataCollectionTest.php

```diff
@@ -82,7 +82,9 @@ private function getUserAgentFixtures()
      */
     public function testSetGetLogger()
     {
-        $logger = $this->getMock('\Monolog\Logger', [], [], '', false);
+        $logger = $this->getMockBuilder(\Monolog\Logger::class)
```

Comment:
`createMock` should work just fine here - as it implicitly does `disableOriginalConstructor` anyway